### ISSUE TITLE
gt package issues

### ISSUE THREAD

**Issue by rudeboybert** [moderndive/moderndive_book#165]
I'm getting the following error with the following `gt` package installation code in `index.rmd` `remotes::install_github("rstudio/gt", ref="eff3be7384365a44459691e49b9b740420cd0851")`
```
Error: Failed to install 'gt' from GitHub:
  Failed to install 'sass' from GitHub:
```
Furthermore, there is issue with the code that creates the "Type I vs II error" table in `10-hypothesis-testing.Rmd`
```
tibble(
  verdict = c("Not guilty verdict", "Guilty verdict"),
  `Truly not guilty` = c("Correct", "Type I error"),
  `Truly guilty` = c("Type II error", "Correct")
) %>% 
  gt(rowname_col = "verdict") %>% 
  tab_header(title = "Type I and Type II errors in US trials",
             label="tab:trial-errors-table") %>% 
  tab_row_group(group = "Verdict")   %>% 
  tab_spanner(
    label = "Truth",
    columns = vars(`Truly not guilty`, `Truly guilty`)
  ) %>% 
  cols_align(align = "center") %>% 
  tab_options(table.width = pct(90))
```
Specifically the `label="tab:trial-errors-table"`. 

Since the two "Type I vs II error" tables are the only place we use the `gt` package, @ismayc how would you feel about using screenshot images of the two tables, but leaving in the `gt` that created them in the `.Rmd` with the code chunk set with `eval =FALSE`?

**Comment by ismayc** [comment]
That’s fine with me. This was definitely annoying to get gt to work and get the table to have a label. It would be easier to just make them images with a figure caption.

**Comment by rudeboybert** [comment]
Fixed in #167
